dp_file_location: /home/ugv/yzchen_ws/omen_deploy/UGV_new/
var_s: 3.0
var_theta: 1.0
step_num: 5
step_dt: 0.5
use_de: false

#var_s: 2.0
#var_theta: 1.0
#step_num: 6
#step_dt: 0.5
#use_de: false
